update-v4-readmes.sh

```diff
@@ -1,38 +1,43 @@
 #!/bin/bash
 
-# The following is a 5 column list of org, repo, branch, package, and file.
-# - If the branch is NOT specified, then the README for that project
-#   will be pulled from npmjs.org instead and will reflect the latest
-#   release.
+# The following is a 5 column list of org, repo, branch, file, and module.
+# - If the module is specified, the README for that project will be
+#   pulled from npmjs.org instead and will reflect the latest release.
+#   For scoped packages such as `@loopback/metadata`,  the `/` needs to
+#   be encoded as `%2f`. For example `@loopback%2fmetadata`.
 # - If the branch IS specified, it will be used to fetch the README.md
 #   from the given github repo. If that branch is NOT master, then the
 #   branch name will be appended to the local readme file name.
+#
+# Examples:
+# strongloop loopback-next master packages/metadata/README.md
```

Comment:
I think this will not work because we are not publishing loopback-next to npmjs.org. Unless I am missing something?

Reply:
We only pull from npmjs if the `module` field (last one) is specified.

Reply:
I see. Could you please enhance the comments to explain what is each example showing?